packages/nimble-components/src/rich-text-editor/template.ts

```diff
@@ -6,19 +6,36 @@ import { iconBoldBTag } from '../icons/bold-b';
 import { iconItalicITag } from '../icons/italic-i';
 import { iconListTag } from '../icons/list';
 import { iconNumberListTag } from '../icons/number-list';
+import { errorTextTemplate } from '../patterns/error/template';
+import { iconExclamationMarkTag } from '../icons/exclamation-mark';
 
 // prettier-ignore
 export const template = html<RichTextEditor>`
-    <template>
+    <template
+        aria-disabled="${x => x.disabled}"
+    >
         <div class="container">
-            <section ${ref('editorContainer')} class="editor-container">
+            <section
+                ${ref('editorContainer')}
+                class="editor-container"
+            >
             </section>
-            <section class="footer-section" part="footer-section">
-                <${toolbarTag}>
+            <${iconExclamationMarkTag}
+                severity="error"
+                class="error-icon ${x => (x.scrollbarWidth >= 0 ? 'scrollbar-width-calculated' : '')}"
+                style="--ni-private-rich-text-editor-scrollbar-width: ${x => x.scrollbarWidth}px;"
+            ></${iconExclamationMarkTag}>
+            <section class="footer-section">
+                <${toolbarTag}
+                    ?disabled="${x => x.disabled}"
+                    aria-disabled="${x => x.disabled}"
```

Comment:
Is `nimble-toolbar` handling this `aria-disabled` internally already?

Reply:
Same comment applies for `nimble-toggle-button` as well. If it's already implemented we can stop with just setting `disabled`

Reply:
`nimble-toolbar` employs the FAST toolbar. It appears that the [template of the FAST toolbar](https://github.com/microsoft/fast/blob/6091f89a6a0b8bf3baa4cd4ed2aae1078ba9865b/packages/web-components/fast-foundation/src/toolbar/toolbar.template.ts#L21C9-L36C10) does not contain `disabled` or `aria-disabled` attributes. However, I believe we should apply the disabled value to its child elements i.e. the toggle buttons. 

Also, the `nimble-toggle-button` internally designates the `aria-disabled` state as true when the `disabled` attribute is set. Therefore, I have also removed it from here.